src/commands/help.ts

```diff
@@ -0,0 +1,8 @@
+import {Message} from 'discord.js'
+
+module.exports = {
```

Comment:
[Can you rewrite this to follow Typescript's export system?](https://www.typescriptlang.org/docs/handbook/modules.html)